pkg/util/removeall/removeall.go

```diff
@@ -25,16 +25,16 @@ import (
 	"k8s.io/mount-utils"
```

Comment:
who is the right sig to own this package? it looks like everything under pkg/util bubbles to pkg/owners, which doesn't seem right

Reply:
sig-storage should be the right sig to own the `removeall` package. But it is just one file.

Reply:
cc @msau42 @jsafrane does sig-storage being approver for `removeall` package seems reasonable ?

Reply:
only storage related logic appears to use this function so I think sig-storage as owners makes sense